configure

```diff
@@ -7,20 +7,20 @@ KOKKOS_LIB=$KOKKOS_PATH/lib64
 ./wcb configure --prefix=$INSTALL \
 --with-tbb="$TBBROOT" \
 --with-jsoncpp="$JSONCPP_FQ_DIR" \
---with-jsonnet="$JSONNET_FQ_DIR" \
---with-eigen-include="$EIGEN_DIR/include/eigen3" \
---with-root="$ROOTSYS" \
+--with-jsonnet="/sdcc/u/kyu/lib/jsonnet/jsonnet" \
```

Comment:
Please restore this file to its original. These variable names are consistent with env variables used in the container. You may have your own version as a local file for convenience. Or you can set the variable names as these which I think is maybe better.